.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
     - $HOME/.ivy2/cache
     - $HOME/.sbt/boot
 
+script:
+  - sbt -J-XX:ReservedCodeCacheSize=256m +test:compile 'set concurrentRestrictions in Global += Tags.limit(Tags.Test, 1)' test
```

Comment:
do we really want to? We have no idea if they'll pass on travis, they're heavily timing sensitive.